app/services/change_risk_validation.rb

```diff
@@ -0,0 +1,51 @@
+class ChangeRiskValidation
+  include Interactor
+  include EntitiesToNotify
+
+  delegate :investigation, :risk_validated_at, :risk_validated_by, :is_risk_validated, :user, to: :context
+
+  def call
+    context.fail!(error: "No investigation supplied") unless investigation.is_a?(Investigation)
+    context.fail!(error: "No user supplied") unless user.is_a?(User)
+
+    context.changes_made = false
+
+    update_risk_validation_attributes
```

Comment:
```suggestion
    assign_risk_validation_attributes
```
I think this is more descriptive of the what the method does. `update` implies it is saved to the database